src/storage-types/src/read_holds.rs

```diff
@@ -8,13 +8,21 @@
 // by the Apache License, Version 2.0.
 
 use std::fmt::Debug;
+use std::sync::Arc;
 
 use mz_repr::GlobalId;
 use thiserror::Error;
 use timely::progress::{Antichain, ChangeBatch, Timestamp as TimelyTimestamp};
 use timely::PartialOrder;
+use tokio::sync::mpsc::error::SendError;
 use tokio::sync::mpsc::UnboundedSender;
 
+type ChangeTx<T> = Box<
```

Comment:
Should we make this an `Arc` directly instead of having an arc wrapping a box?

Reply:
💯